app/views/personas/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :page_title, I18n.t("components.page_titles.sign_in.personas") %>
+
+<h1 class="govuk-heading-xl">
+  <span class="govuk-caption-xl">
+    Teacher Training API
+  </span>
+  Personas
+</h1>
+
+<hr class="govuk-section-break govuk-section-break--visible govuk-section-break--l">
+
+<h2 class="govuk-body">
+
+  <p class="govuk-body"><%= govuk_tag(text: "Admin", colour: "blue") %></p>
+
+  <p class="govuk-body">
+     Colin is a DfE support agent who has administrator access to all organisations.
+  </p>
+</h2>
+
+<%= render Personas::View.new(email_address: Settings.personas.admin.email_address,
```

Comment:
I'm not sure we really need these all as changeable settings.. as a developer I don't really need the ability to change 'Colin' to 'Colinetta' for example. They might better live in `en.yml` if you don't want to have strings directly in the view.

If you're going to do that, may as well move the rest of the strings in this file to `en.yml`!